src/dmd/aggregate.d

```diff
@@ -847,6 +847,12 @@ extern (C++) abstract class AggregateDeclaration : ScopeDsymbol
         return type;
     }
 
+    // Does this class have an invariant function?
+    final bool hasInvariant()
```

Comment:
can be `const`, and all the other `hasInvariants()` too. Excepted perhaps the one that may cache the information.

Reply:
Won't work because of the overriding one that can't be const.